src/components/Reports/StaffExpenseReport/BalanceCard/BalanceCard.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Visibility } from '@mui/icons-material';
+import { Box, Card, CardActionArea, Typography, styled } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Fund } from 'src/graphql/types.generated';
+
+interface BalanceCardProps {
+  fundType: Fund['fundType'];
+  title: string;
+  icon: React.ComponentType;
+  iconBgColor?: string;
+  startingBalance: number;
+  endingBalance: number;
+  transfersIn: number;
+  transfersOut: number;
+  onClick?: (fundType: Fund['fundType']) => void;
+  isSelected?: boolean;
+}
+
+const ScreenOnly = styled(Box)(() => ({
+  '@media print': {
+    display: 'none',
+  },
+}));
+
+export const BalanceCard: React.FC<BalanceCardProps> = ({
+  fundType,
+  title,
+  icon: Icon,
+  iconBgColor,
+  startingBalance,
+  endingBalance,
+  transfersIn,
+  transfersOut,
+  onClick,
+  isSelected = false,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card
+      variant="outlined"
+      sx={{
+        p: 2,
+        flex: 1,
+        minWidth: 0,
+        maxWidth: 'none',
+        fontSize: '1.25rem',
+        boxShadow: isSelected ? 3 : 1,
+        transition: 'box-shadow 0.3s ease-in-out',
+      }}
+    >
+      <Box display={'flex'} flexDirection="row" alignItems="center" gap={1}>
+        <Box
+          sx={{
+            backgroundColor: iconBgColor || 'primary.main',
+            color: 'primary.contrastText',
+            borderRadius: 1,
+            p: 1,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+          }}
+        >
+          <Icon />
+        </Box>
+        <Typography variant="h6">{title}</Typography>
+      </Box>
+      <Box display="flex" flexDirection="column" mt={3} mb={2}>
+        <Typography>
+          {t('Starting Balance: ')}
+          {startingBalance !== undefined
+            ? startingBalance.toLocaleString(undefined, {
```

Comment:
I think we want to get the `locale` from `useLocale()` and use it here so that it formats the amounts using the user's locale preference.

```suggestion
            ? startingBalance.toLocaleString(locale, {
```